qubes/mgmt.py

```diff
@@ -534,3 +570,95 @@ def vm_kill(self):
         assert not self.arg
         self.fire_event_for_permission()
         yield from self.dest.kill()
+
+    @api('mgmt.Events', no_payload=True)
```

Comment:
We want the protocol as simple as possible. Surely we don't want to have interactive protocol here. And requiring EOF before receiving anything (and actually triggering the operation) seems like a straight forward solution - this way we're sure there wont be additional input coming in.
Re "socket interface" - keep in mind _the same_ should also work over qrexec.